sqlmesh/dbt/builtin.py

```diff
@@ -164,6 +164,58 @@ def has_var(self, name: str) -> bool:
         return name in self.variables
 
 
+class ConfigObject:
```

Comment:
Nit: let's rename to just `Config` for brevity & consistency w/ others in `builtin.py`.